source/contracts/reporting/Reporting.sol

```diff
@@ -19,6 +19,7 @@ library Reporting {
     uint256 private constant BOND_PAYOUT_MULTIPLIER = 2; // Multiplier applied to the dispute bond payment to get what they should ideally be compensated with
     uint256 private constant DEFAULT_REPORTING_FEE_DIVISOR = 100; // 1% fees
     uint256 private constant MAXIMUM_REPORTING_FEE_DIVISOR = 10000; // Minimum .01% fees
+    uint256 private constant MINIMUM_REPORTING_FEE_DIVISOR = 3; // Maximum 33.3~% fees
```

Comment:
Recommend enhancing this comment with a mention that any divisor lower than 2 _may_ result in bugs like divide-by-zero.  I don't want someone coming in here and adjusting it to 50% thinking that is a "reasonable default" and introducing a crasher.